src/Microsoft.Health.Fhir.Core/Features/Search/Registry/ResourceSearchParameterStatus.cs

```diff
@@ -0,0 +1,20 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Health.Fhir.Core.Features.Search.Registry
+{
+    public class ResourceSearchParameterStatus
+    {
+        public Uri Uri { get; set; }
+
+        public SearchParameterStatus Status { get; set; }
+
+        public bool? IsPartiallySupported { get; set; }
```

Comment:
What is the benefit of using the nullable boolean? It looks like you only set it to 'true' or don't set it.

Reply:
Original thought is that I only know explicitly when Partial Support is true. The inverse is unset / unknown but not explicitly false, perhaps I'm over thinking it.